SPECS/netplan/netplan.spec

```diff
@@ -7,103 +7,241 @@
 # If this isn't defined, define it
 %{?!_systemdgeneratordir:%global _systemdgeneratordir /usr/lib/systemd/system-generators}
 
-# Force auto-byte-compilation to Python 3
-%global __python %{__python3}
+# Netplan library soversion major
+%global libsomajor 0.0
 
 
 Name:           netplan
-Version:        0.95
+Version:        0.107.1
 Release:        1%{?dist}
 Summary:        Network configuration tool using YAML
 Group:          System Environment/Base
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 License:        GPLv3
 URL:            https://netplan.io/
-# Source0:      https://github.com/canonical/%{name}/archive/%{version}/%{version}.tar.gz
-Source0:        %{name}-%{version}.tar.gz
+Source0:        https://github.com/canonical/%{name}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
Not sure about this `#/%{name}-%{version}.tar.gz` appended at the end, is this intended or a left-over comment?

Reply:
this typically renames the file during download ... in this case, prefixing the package name to the tarball.